code/data_structures/stack/reverse_stack/reverse_stack.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
A stack should have functions such as `pop()` and `push()`. It would be best if you adher to OOP design. [See this](https://github.com/OpenGenus/cosmos/blob/master/code/data_structures/stack/stack/Stack.php) for guidance 👍

Reply:
thnx bro @AdiChat